### DISCUSSION TITLE
Can axolotl generate LoRAs for Quip# models?

### DISCUSSION THREAD

**Discussion by araleza** [axolotl-ai-cloud/axolotl#1006] [Q&A] [answered]
Hi,

Like many people, I have 24GB of VRAM.  I've read good things about Quip# and its ability to load a 70B model onto a 24GB card.  Is it possible to generate a LoRA for them using axolotl?  I couldn't see anything that looked appropriate in the examples directory.  Is support for these models planned?

Thanks...

**Comment by NanoCode012** [comment, marked as answer]
I'm not sure which model you mean, but, if it's this, you can see that it's just a Mistral model. https://huggingface.co/abhishek/quip-v1/blob/main/config.json

You can use the mistral examples.
